lib/intent-models/channel.js

```diff
@@ -0,0 +1,67 @@
+
+module.exports = {}; 
+const IntentBase = require('./base');
+// eslint-disable-next-line no-unused-vars
+const Schema = require('../models/schema');
+const { IntentMessage } = require('./message');
+const { IntentOperation } = require('./operation');
+const { IntentChannelParameter } = require('./channel-parameter');
+
+/**
+ * 
+ * @class
+ * @alias module:@asyncapi/parser#IntentChannel
+ * @extends IntentBase
+ */
+class IntentChannel extends IntentBase {
+  /**
+   * @returns {IntentOperation[]}
+   */
+  operations() { return [new IntentOperation()];}
```

Comment:
Should we include all the intents from the root in terms of the point of view for operations, such as `applicationPublishOperations` and `applicationSubscribeOperations`?

Reply:
I think I will leave it for the next version ?

Reply:
Alright 👍